src/Node/FS.purs

```diff
@@ -7,6 +17,32 @@ import Prelude
 --
 foreign import data FS :: !
 
+foreign import data FileDescriptor :: *
+
+data FileFlags = R | R_PLUS | RS | RS_PLUS
+               | W | WX | W_PLUS | WX_PLUS
+               | A | AX | A_PLUS | AX_PLUS
+
+instance showFileFlags :: Show FileFlags where
```

Comment:
I don't really mind show used this way, but I'll change the code to suit. Presumably you'll want to fix this also:

https://github.com/purescript-node/purescript-node-buffer/blob/master/src/Node/Encoding.purs#L17